x-pack/plugins/security/public/management/role_mappings/edit_role_mapping/rule_editor_panel/json_rule_editor.tsx

```diff
@@ -5,6 +5,7 @@
  * 2.0.
  */
 
+import 'react-ace';
```

Comment:
> `react-ace` has been added to the `kbn-ui-shared-deps` package to mitigate plugin bundle size increases. This is _more similar_ to how it was included in Kibana when it was part of EUI.

I'm not sure what this import is doing. Is it actually necessary, or was it perhaps an accident?

Reply:
It was necessary when I added it (although that was before https://github.com/elastic/kibana/pull/117242/commits/5c3ba0aa5d6477a5c2bb461196f31fc6188f3c79) - the issue is that without it, the `import 'brace/*'` will cause an error with an `ace is not defined` when Jest tests that import the file are run. You can see https://github.com/elastic/kibana/runs/4110664841 for the list of failures.